projectcard/schema/transit/transit_route.json

```diff
@@ -37,11 +37,13 @@
         ]
       }
     },
+    "required": ["agency_id", "route_type"],
```

Comment:
`route_id` should be required, that is something the user can code in the CUBE line name and Lasso can write out to project card.

Reply:
ok - wasn't sure if we were going to generate it. added now as required.